src/main/java/com/epam/reportportal/cucumber/AbstractReporter.java

```diff
@@ -278,34 +294,54 @@ private void removeFromTree(RunningContext.FeatureContext featureContext, Runnin
 	 */
 	@Nonnull
 	@SuppressWarnings("unused")
-	protected FinishTestItemRQ buildFinishTestItemRequest(@Nonnull Maybe<String> itemId, @Nullable ItemStatus status) {
+	protected FinishTestItemRQ buildFinishTestItemRequest(@Nonnull Maybe<String> itemId, @Nullable ItemStatus status,
+														  @Nullable Throwable error) {
 		FinishTestItemRQ rq = new FinishTestItemRQ();
+		if (status == ItemStatus.FAILED) {
+			Optional<String> currentDescription = Optional.ofNullable(descriptionsMap.get(itemId.blockingGet()));
+			currentDescription.flatMap(description -> Optional.ofNullable(error)
+							.map(errorMessage -> resolveDescriptionErrorMessage(description, errorMessage)))
+					.ifPresent(rq::setDescription);
+		}
 		ofNullable(status).ifPresent(s -> rq.setStatus(s.name()));
 		rq.setEndTime(Calendar.getInstance().getTime());
 		return rq;
 	}
 
+	/**
+	 * Resolve description
+	 * @param currentDescription Current description
+	 * @param error Error message
+	 * @return Description with error
+	 */
+	private String resolveDescriptionErrorMessage(String currentDescription, Throwable error) {
+		return Optional.ofNullable(currentDescription)
+				.filter(StringUtils::isNotBlank)
+				.map(description -> format(DESCRIPTION_ERROR_FORMAT, currentDescription, error))
+				.orElse(format(ERROR_FORMAT, error));
+	}
+
 	/**
 	 * Finish a test item with specified status
 	 *
 	 * @param itemId an ID of the item
 	 * @param status the status of the item
 	 */
-	protected void finishTestItem(@Nullable Maybe<String> itemId, @Nullable ItemStatus status) {
+	protected void finishTestItem(@Nullable Maybe<String> itemId, @Nullable ItemStatus status, @Nullable Throwable error) {
```

Comment:
Please do not modify existing method signatures, they are part of API. Overload with inner method calling if you need new one.

Reply:
The issue was fixed in commit b4c87fd1b8efa718a2e6076370b40a3b1e3274c8
Thank you !